suite-native/module-passphrase/src/passphraseThunks.ts

```diff
@@ -56,3 +57,14 @@ export const verifyPassphraseOnEmptyWalletThunk = createThunk(
         return fulfillWithValue(true);
     },
 );
+
+export const retryPassphraseAuthentication = createThunk(
+    `${PASSPHRASE_MODULE_PREFIX}/retryPassphraseAuthentication`,
+    (_, { dispatch, getState }) => {
+        const device = selectDevice(getState());
+
+        if (!device) return;
+        dispatch(deviceActions.removeButtonRequests({ device }));
```

Comment:
Is it necessary to delete all button request? the `removeButtonrequests` action has second argument `buttonRequestCode`. Isn't it enough to delete only the passphrase request? Not sure just thinking if this call will not clash with other device button request features.

Reply:
It's actually not necessary per say, but it's a precaution in case there would be some hanging button requests. Not sure whether this can happen but it certainly doesn't hurt. Since we're about to re-authorize the device, let's bomb any button requests available at this moment to have a clean state when we start the authorization process. I can of course remove it, it's just in case kind of thing here..not sure.

Reply:
Ok, let's leave it for now. If this issue occurs, we can change it later.